addons/account/migrations/12.0.1.1/post-migration.py

```diff
@@ -192,8 +192,8 @@ def fill_account_move_reverse_entry_id(env):
 def recompute_invoice_taxes_add_analytic_tags(env):
     analytic_taxes = env['account.tax'].search([('analytic', '=', True)])
     taxes = env['account.tax'].search([
-        ('id', 'parent_of', analytic_taxes.ids),
-    ])
+        ('children_tax_ids', 'in', analytic_taxes.ids),
```

Comment:
Not sure if this operator is going to behave as expected. Have you tried it? Does it mean "one of the records in `children_tax_ids` is one of the `analytic_taxes`?

Reply:
YES

Reply:
master, believe in this young padawan